studio/components/interfaces/Billing/ProUpgrade.tsx

```diff
@@ -290,32 +288,24 @@ const ProUpgrade: FC<Props> = ({
                       <SupportPlan currentOption={currentAddons.supportPlan} />
                     </>
                   )}
-                  {isCustomDomainsEnabled && customDomainOptions.length > 0 && (
```

Comment:
Just confirming we no longer need the check for `customDomainOptions.length > 0` as well?

Reply:
yup! these were there with the assumption that the API might not return those options yet